### PR TITLE
feat: develop labs page

### PR DESCRIPTION
**Pull request by nikvoblikov**
**This PR:**
- Adds a page with labs;
- Adds labs to the header and footer.

TODO before merge:
- [x] Remove test labs.

**Steps to test**

- Open the [Labs page](https://deploy-preview-438--ebpfio.netlify.app/labs/)
- Confirm that everything looks and works as expected

### CODE REVIEW

File: src/components/pages/labs/categories/categories.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const Categories = ({ categories, currentCategory, className }) => {
+  const handleCategoryClick = (slug) => (event) => {
+    event.preventDefault();
+    navigate(slug, {
+      state: { preventScroll: true },
+    });
+  };
+
+  return (
+    <section className={clsx('container-sm', className)}>
```
Comment (vannyle) [inline src/components/pages/labs/categories/categories.jsx:15]: In the section, there should be the class `safe-paddings`. And only in the next div element we leverage `container`. Please see other components for examples (e.g. [home page](https://deploy-preview-438--ebpfio.netlify.app/))
image

File: src/components/pages/labs/categories/categories.jsx [hunk 2]
```diff
@@ -1,33 +1,52 @@
...
+  useEffect(() => {
+    const activeIndex = categories.findIndex(({ name }) => name === currentCategory);
+    if (activeIndex !== -1 && categoryRefs.current[activeIndex]) {
+      categoryRefs.current[activeIndex].scrollIntoView({
+        behavior: 'smooth',
+        block: 'nearest',
+      });
+    }
+  }, [categories, currentCategory]);
+
```
Comment (vannyle) [inline src/components/pages/labs/categories/categories.jsx:23]: This function doesn't work for me either. I'd suggest removing it for now, and trying to test the API https://www.gatsbyjs.com/docs/reference/config-files/gatsby-browser/#shouldUpdateScroll

File: src/templates/labs.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    }
+    navigate(navigatePath);
+  };
+
+  return (
+    <Layout>
+      <section className="safe-paddings">
+        <div className="container flex flex-col pt-20 lg:pt-16">
+          <h1 className="heading-9xl mx-auto max-w-[890px] text-center font-semibold leading-tight">
+            Dig into the eBPF with our interactive labs
```
Comment (xmulligan) [inline src/templates/labs.jsx:36]:
```suggestion
            Dig into eBPF with interactive labs
```

**Comment by nikvoblikov** [comment]
> Can we remove the extra labs and have the getting started lab under just "programming". The tetragon lab should also be removed because it isn't focused enough on just eBPF.
> 
> We are waiting for the development of a second eBPF lab before we want to merge this PR. Until we have more labs, can we also comment out the filters at the top? Then we can just add them back in later when we have labs for each of those sections.

Done, I set the condition that if there is more than one category, then filters will appear.

**Comment by xmulligan** [comment]
Thanks, I'm going to wait to merge this until we have the second ebpf lab done, but otherwise this looks ready to ship!

**Comment by xmulligan** [comment]
@nikvoblikov can you also add this lab https://isovalent.com/labs/learning-ebpf-tutorial/ and resolve the conflict? Then we should be ready to ship it

**Comment by xmulligan** [comment]
@nikvoblikov friendly ping on this so we can get it merged :)

**Comment by nikvoblikov** [comment]
> @nikvoblikov we would like this merged before eBPF Summit wednesday if possible

Hi, @xmulligan I made a rebase, resolve conflicts and added a new lab.

**Review by xmulligan — APPROVED** [review]

**Pull request merged**